source/routing/index.md

```diff
@@ -1,32 +1,22 @@
-## Routing
+Imagine we are writing a web app for managing a blog. At any given time, we
+should be able to answer questions like _What post are they looking at?_ and
+_Are they editing it?_ In Ember.js, the answer to these questions is determined
+by the URL.
 
-As users interact with your application, it moves through many
-different states. Ember.js gives you helpful tools for managing
-that state in a way that scales with your application.
+The URL can be set in a few ways:
 
-To understand why this is important, imagine we are writing a web app
-for managing a blog. At any given time, we should be able to answer
-questions like: _Is the user currently logged in? Are they an admin
-user? What post are they looking at? Is the settings screen open?  Are
-they editing the current post?_
+* The user loads the app for the first time.
+* The user changes the URL manually, such as by clicking the back button or by
+editing the address bar.
+* The user clicks a link within the app.
+* Some other event in the app causes the URL to change.
 
-In Ember.js, each of the possible states in your application is
-represented by a URL. Because all of the questions we asked above—
-_Are we logged in?  What post are we looking at?_ —are encapsulated by
-route handlers for the URLs, answering them is both simple and accurate.
+Regardless of how the URL becomes set, the Ember router then maps the current
+URL to a route handler. The route handler can do several things:
 
-At any given time, your application has one or more _active route
```

Comment:
I think the bit about one or more active route handlers should be in, somewhere. Is it mentioned later?